packages/playground/src/domains/agent-builder/components/agent-edit/agent-profile/__tests__/agent-profile-model-step.test.tsx

```
@@ -0,0 +1,116 @@
+// @vitest-environment jsdom
+import { MastraReactProvider } from '@mastra/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { cleanup, render, screen } from '@testing-library/react';
+import { http, HttpResponse } from 'msw';
+import type { ReactNode } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { MemoryRouter } from 'react-router';
+import { afterEach, describe, expect, it } from 'vitest';
+
+import { server } from '@/test/msw-server';
```

ESLint

[error] `@/test/msw-server` import should occur after import of `./fixtures/agent-profile-tabs`

(import/order)